sdks/python/test-suites/direct/common.gradle

```diff
@@ -56,6 +56,31 @@ tasks.register("postCommitIT") {
   }
 }
 
+tasks.register("xlangAwsIOIT") {
```

Comment:
You can automate this part using our existing utils. Steps:

1. Create a new pytest marker [here](https://github.com/apache/beam/blob/0536dc6f042c8e6ecf920e9b92671cbb8e015eda/sdks/python/pytest.ini#L32)
2. Mark your new test in `xlang_sqsio_it_test.py` with pytest.mark.<new-marker>
3. Follow these instructions: https://github.com/apache/beam/blob/0536dc6f042c8e6ecf920e9b92671cbb8e015eda/sdks/python/test-suites/xlang/build.gradle#L24-L25

Reply:
If you name the new task "awsCrossLanguage" for example, it will create a new gradle task `":sdks:python:test-suites:direct:py<version>:awsCrossLanguagePythonUsingJava"` that you can reference in `sdks/python/test-suites/direct/build.gradle`